plugin/src/test/java/io/jenkins/plugins/casc/SchemaGenerationTest.java

```diff
@@ -47,4 +47,20 @@ public void downloadOldSchema() throws Exception {
         );
         writer.close();
     }
+
+
+    @Test
+    public void validateSchema() {
+
+    }
+
+    @Test
+    public void checkRootConfigurators() {
```

Comment:
@casz @oleg-nenashev would it be a good thing if we check the schema against individual configurators.
Considering we are using two configurators
a) RootConfigurators
b) BaseConfigurators
c) HeteroDescribableConfigurators.